src/python/foglamp/translators/omf_translator.py

```diff
@@ -0,0 +1,732 @@
+                        "format": "date-time",
+                        "type": "string",
+                        "isindex": True
+                    },
+                    "x": {
+                        "type": "number"
+                    },
+                    "y": {
+                        "type": "number"
+                    },
+                    "z": {
+                        "type": "number"
+                    },
+                    "pressure": {
+                        "type": "integer"
+                    },
+                    "lux": {
+                        "type": "integer"
+                    },
+                    "humidity": {
+                        "type": "number"
+                    },
+                    "temperature": {
+                        "type": "number"
+                    },
+                    "object": {
+                        "type": "number"
+                    },
+                    "ambient": {
+                        "type": "number"
+                    }
+
+                }
+            }
+        ]
+
+
+    except Exception as e:
+        status  = False
+        message = _message_list["e000006"].format(e)
+
+        _log.error(message)
+        raise Exception(message)
+
+    return status
+
+
+def debug_msg_write(severityMessage, message):
+    """Writes a debug message
+    """
+
+    if LOG_SCREEN == True:
+
+        if severityMessage == "":
+            print("{0:}".format(message))
+        else:
+            print ("{0:} - {1:<7} - {2} ".format(time.strftime("%Y-%m-%d %H:%M:%S:"), severityMessage ,message) )
+
+    _log.debug(message)
+
+    return
+
+
+def create_data_values_stream_message(target_stream_id, information_to_send):
+    """Creates the JSON data for OMF
+
+    Args:
+        target_stream_id:     OMF container ID
+        information_to_send:  information retrieved that should be prepared for OMF
+
+    Returns:
+        status: True if successful, False otherwise.
+        data_values_JSON: information converted in JSON format
+
+    Raises:
+        Exception: unable ro create the JSON message.
+
+    """
+
+    status = True
+    data_available = False
+
+    data_values_JSON = ''
+
+    row_id      = information_to_send.id
+    asset_code  = information_to_send.asset_code
+    timestamp   = information_to_send.user_ts.isoformat()
+    sensor_data = information_to_send.reading
+
+    debug_msg_write("INFO", "Stream ID : |{0}| ".format(target_stream_id))
+    debug_msg_write("INFO", "Sensor ID : |{0}| ".format(asset_code))
+    debug_msg_write("INFO", "Row    ID : |{0}| ".format(str(row_id)))
+
+    try:
+        # Prepares the data for OMF
+        data_values_JSON = [
+            {
+                "containerid": target_stream_id,
+                "values": [
+                    {
+                        "Time": timestamp
+                    }
+                ]
+            }
+        ]
+
+        #
+        # Evaluates which data is available
```

Comment:
I'm not sure what you are trying to achieve here, is it to make sure the reading contains a tag that matches the properties in the type information defined within our container?